packages/webrtc-core/src/media/track.ts

```diff
@@ -171,7 +171,7 @@ export abstract class Track extends EventEmitter<TrackEvents> {
    * @returns #mediaStreamTrack of type MediaStreamTrack.
    */
   getMediaStreamTrackWithEffects(): MediaStreamTrack | undefined {
```

Comment:
Since `#mediaStreamTrack` is required, we can remove `undefined` from the signature now.

Reply:
will remove it in the next PR waiting for brian comment

Reply:
done